how-to/workspace-platform-starter/public/manifest.fin.json

```diff
@@ -447,6 +447,42 @@
 						}
 					}
 				},
+				{
+					"include": true,
```

Comment:
shouldn't this be false. i.e. don't include button?

Reply:
If the condition module is not enabled it won't show anyway, but I could set it to false.